<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKxFilterListView</class>
 <widget class="QWidget" name="QKxFilterListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="panel">
     <item>
      <widget class="QPushButton" name="btnSSH">
       <property name="text">
        <string>SSH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSFTP">
       <property name="text">
        <string>SFTP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTelnet">
       <property name="text">
        <string>Telnet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVNC">
       <property name="text">
        <string>VNC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="woterm.qrc">
         <normaloff>../private/skins/black/add2.png</normaloff>../private/skins/black/add2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="woterm.qrc"/>
 </resources>
 <connections/>
</ui>
